src/admin/components/OnlineDriver/OnlineDriverTable.tsx

```diff
@@ -1,20 +1,12 @@
 import { useEffect, useState } from "react";
-
-import { IOnlineDriver } from "../../../api/type";
-import { OnlineDriverApi } from "../../../api/OnlineDriverApi";
-import { OnlineDriverRepo } from "../../../repositories/OnlineDriverRepo";
-
-
-
+import { useRepository } from "../../../hooks/CustomHook";
 
 function OnlineDriverTable() {
-  
+  const { repo } = useRepository();
   const [Onlinedrivers, setOnlineDrivers] = useState<OnlineDriver[]>([]);
   useEffect(() => {
     const fetchData = async () => {
```

Comment:
Let's move this `fetchData` method to "actions/fetchOnlineDriverData" or similar and use that method wherever necessary.